pkg/clientconn/roxctl_command_header.go

```diff
@@ -0,0 +1,9 @@
+package clientconn
+
+// RoxctlCommandHeader is the HTTP header name with the reconstructed
+// roxctl command as the value.
+const RoxctlCommandHeader = "Rh-Roxctl-Command"
+
+// RoxctlCommandIndexHeader is the HTTP header name with the 1-based index
```

Comment:
I don't understand it :(

Reply:
Maybe something like "represents the sequential order of API call for given command."

Reply:
I updated the comment. Please check if you like it better.